x-pack/elastic-agent/pkg/composable/providers/agent/agent.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/agent/errors"
 	"github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/composable"
 	"github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/config"
+	corecomp "github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/core/composable"
```

Comment:
Why this adjustment everywhere?

Reply:
Otherwise it will hit circular import errors on compile.